### PR TITLE
Update for hive ocm-4.4.0 rename to ocm-1.0.

### PR DESCRIPTION
**Pull request by dgoodwin**

### CODE REVIEW

File: ci-operator/jobs/openshift/hive/openshift-hive-ocm-1.0-presubmits.yaml [hunk 1]
```diff
@@ -84,9 +84,9 @@ presubmits:
           secretName: regcred
     trigger: (?m)^/test( | .* )e2e,?($|\s.*)
   - agent: kubernetes
-    always_run: false
+    always_run: true
```
Comment (staebler) [inline ci-operator/jobs/openshift/hive/openshift-hive-ocm-1.0-presubmits.yaml:87]: I don't think that we want the e2e-azure test to always run.

File: ci-operator/jobs/openshift/hive/openshift-hive-ocm-1.0-presubmits.yaml [hunk 2]
```diff
@@ -95,8 +95,7 @@ presubmits:
     labels:
       ci-operator.openshift.io/prowgen-controlled: "true"
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
-    name: pull-ci-openshift-hive-ocm-4.4.0-e2e-azure
-    optional: true
```
Comment (staebler) [inline ci-operator/jobs/openshift/hive/openshift-hive-ocm-1.0-presubmits.yaml:99]: This should be optional.

File: ci-operator/jobs/openshift/hive/openshift-hive-ocm-1.0-presubmits.yaml [hunk 3]
```diff
@@ -168,9 +167,9 @@ presubmits:
           secretName: regcred
     trigger: (?m)^/test( | .* )e2e-azure,?($|\s.*)
   - agent: kubernetes
-    always_run: false
+    always_run: true
```
Comment (staebler) [inline ci-operator/jobs/openshift/hive/openshift-hive-ocm-1.0-presubmits.yaml:170]: false

File: ci-operator/jobs/openshift/hive/openshift-hive-ocm-1.0-presubmits.yaml [hunk 4]
```diff
@@ -179,8 +178,7 @@ presubmits:
     labels:
       ci-operator.openshift.io/prowgen-controlled: "true"
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
-    name: pull-ci-openshift-hive-ocm-4.4.0-e2e-gcp
-    optional: true
```
Comment (staebler) [inline ci-operator/jobs/openshift/hive/openshift-hive-ocm-1.0-presubmits.yaml:183]: optional

**Comment by dgoodwin** [comment]
Updated, thanks!

**Pull request merged**